examples/flink-sql-runner-example/README.md

```diff
@@ -0,0 +1,77 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Flink Kubernetes Operator SQL Example
+
+## Overview
+
+This is an end-to-end example of running Flink SQL scripts directly using the Flink Kubernetes Operator.
+
+SQL script submission might one day become a top level feature of the operator but due to current limitations some manual work is required to customise the logic for the given user environment.
```

Comment:
Since we are not planning to include any sql runner logic into the operator baseline due to the dependency issues it would bring along, should we set the expectation that this is NOT going to be an operator feature? If Flink one day has a SQL support for application mode, then there should hopefully be nothing extra required in the operator.

Reply:
Sounds reasonable, I will change the wording to not give the wrong impression